src/themes/dspace/app/submission/sections/upload/section-upload.component.html

```diff
@@ -0,0 +1,43 @@
+<ds-alert [content]="'submission.sections.upload.info'"
+          [dismissible]="true"
+          [type]="AlertTypeEnum.Info"></ds-alert>
+
+<ng-container *ngIf="fileList.length == 0">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-center"><span class="text-muted">{{'submission.sections.upload.no-file-uploaded' | translate}}</span></h3>
```

Comment:
span seems unecessary --> `<h3 class="text-center text-muted"> ...</h3>`

Reply:
not my code :/